### ISSUE TITLE
[Feature request] Allow an Android device to scan a Ubuntu one

### ISSUE THREAD

**Issue by ghost** [janimo/textsecure-qml#41]
Hi, thank you for your work!
I was wondering if you could add the possibility for the Ubuntu app of Signal to display a QR, allowing an Android phone to scan it and pair it like it were a desktop device. I feel like this may be very well appreciated especially since [Ubuntu tablets are coming](http://www.ubuntu.com/tablet/devices) :)
Thank you!

Related to #36

**Comment by janimo** [comment]
This would work if the Ubuntu app were only such a slave device, and not one with its own phone number as it is now. This is useful in the future though, when the Ubuntu app works well enough as a desktop client.

**Comment by M13Joe** [comment]
Dear Janimo, thanks for your work. I just have such an Ubuntu Tablet (BQ Aquaris M10) and cannot receive an SMS (text) for confirmation. I have registered previously for Signal with my Android mobile phone using the call option since the SMS confirmation doesn't work. That may be an issue due to my service provider. Thus, I cannot register to your app via SMS received on my phone. I'd be very happy if you added a call option. Otherwise, a QR code as mentioned by ghost would be fine as well.
Thanks in advance!

**Comment by janimo** [comment]
Yes, voice confirmation would be good to have, several people encounter this blocker. Not sure when I can get around to this though.
